neon-animation-runner-behavior.html

```diff
@@ -32,7 +32,7 @@
             try {
               result = neonAnimation.configure(config);
               // Check if we have an Effect rather than an Animation
-              if (typeof result.cancel != 'function') { 
+              if (!result || typeof result.cancel != 'function') { 
```

Comment:
why this addition? is `null | undefined | 0 | ""` a legit value for `document.timeline.play`?